controllers/articles-controller.js

```diff
@@ -23,4 +26,23 @@ const sendIndividualArticle = (req, res, next) => {
     });
 };
 
-module.exports = { sendArticles, sendIndividualArticle };
+const sendArticleComments = (req, res, next) => {
+  const { article_id } = req.params;
+
+  // promise variables
+  const selectArticleCommentsPromise = selectArticleComments(article_id);
+  const articleExistsPromise = articleExists(article_id);
+
+  Promise.all([selectArticleCommentsPromise, articleExistsPromise])
```

Comment:
This should (maybe) allow your 404 test to work straight away. It will differentiate between valid article with no comments: rows = [], and invalid article: rows = []